packages/core/src/model.ts

```diff
@@ -107,6 +127,7 @@ export function createModel(initialContext: object, creators?): unknown {
           type: eventType
         }))
       : undefined) as any,
+    actions: null as any,
```

Comment:
this ain't finished just yet, right? this can somewhat easily support custom actions - what about built-in ones? how to model, for example, `send` with it?

Reply:
Yes, we should ideally support built-in ones too. Not sure if we should do this via actually baking them in (wanting to be cognizant of bundle size) or something like:

```ts
actions: someModel.action(send('TYPE_SAFE_EVENT'))
```

So that we can pass in "bare" actions but they'll be type-safe

Reply:
> actions: someModel.action(send('TYPE_SAFE_EVENT'))

What kind of type-safety would this give to the user? I don't immediately see a benefit from this PoV. Could you elaborate?

A follow-up question would be - do you intend to support context/event-based payloads with this API?